sources/api/pluto/src/ec2.rs

```diff
@@ -50,8 +51,9 @@ pub(super) async fn get_private_dns_name(region: &str, instance_id: &str) -> Res
 
     let client = if let Some(https_proxy) = https_proxy {
         let http_client = proxy::setup_http_client(https_proxy, no_proxy)?;
+        let http_client = HyperClientBuilder::new().build(http_client);
```

Comment:
This naming is odd to me, can we shift it to not reuse the name, until I realized you were consuming the first `http_client` I wasn't sure if this was intentional.

Reply:
sure